src/planwise/client/scenarios/views.cljs

```diff
@@ -16,16 +16,23 @@
             [planwise.client.utils :as utils]
             [planwise.client.ui.rmwc :as m]))
 
+(defn- show-provider
+  [{{:keys [action name capacity investment]} :elem :as provider}]
+  (if (nil? action)
+    (str "<b>" name "</b><br> Capacity: " capacity)
```

Comment:
Since `name` is user provided, it should be properly escaped to avoid Javascript injection attacts. You can use `planwise.clients.utils/escape-html`.